common/src/main/java/com/wynntils/features/stateless/FixSpellOverwriteFeature.java

```diff
@@ -16,7 +16,7 @@
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-public class FixSpellOverwriteFeature extends InternalFeature {
+public class FixSpellOverwriteFeature extends StatelessFeature {
```

Comment:
Why can't this be a normal user enabled feature? If I understand it correctly, it blocks Wynncraft from updating the held weapon when casting a spell..? Is that because it interferes with other parsing we do, or because of it looking nicer? If it interferes, what is it interfering with?

Reply:
I have no idea, I think p0ke just didn't think about it being disabled. Although it does temporarily break item transformation, and it won't become a gear item stack.

Reply:
If we were to "break" item gear parsing and for example rely spell cast feature on it, it would not be nice.